apps/vr-tests-react-components/src/stories/Positioning.stories.tsx

```diff
@@ -477,6 +477,110 @@ const AutoSize = () => {
   );
 };
 
+const AutoSizeAsyncContent = () => {
```

Comment:
for this test I never see the loading state in the screenshots 
![image](https://github.com/microsoft/fluentui/assets/20744592/89c97d86-261d-4ba7-9745-82f4ba39e0a0)

Reply:
I updated it :)